web/src/lib/cms/previews/profile.tsx

```diff
@@ -54,6 +55,7 @@ const ProfilePreview = (props: Props) => {
           municipalities={[]}
           CMS_ONLY_tab={tab as ProfileTabs}
           CMS_ONLY_businessPersona={businessPersona}
+          CMS_ONLY_foreignBusinessType={foreignBusinessType}
```

Comment:
See comment above. Instead of returning `foreignBusinessType` from getMetadataFromSlug, why not just set this line as `persona == "dakota" ? "NEXUS" : "NONE"`?

This avoids changing the function signature on a widely-used function in support of solving a narrow one-time use case